packages/sit-onyx/src/components/examples/GridPlayground/GridElement/GridElement.ct.tsx

```diff
@@ -0,0 +1,63 @@
+import { expect, test } from "../../../../playwright/a11y";
+import {
+  executeMatrixScreenshotTest,
+  mockPlaywrightIcon,
+} from "../../../../playwright/screenshots";
+import OnyxIcon from "../../../OnyxIcon/OnyxIcon.vue";
+import GridElement from "../GridElement/GridElement.vue";
+
+test.describe("screenshot tests", () => {
+  for (const mode of ["default", "outline"] as const) {
+    executeMatrixScreenshotTest({
+      name: `Grid element (${mode})`,
+      columns: ["default", "with-content"],
+      rows: ["default", "hover", "active", "focus-visible"],
+      component: (column) => (
+        <GridElement columnCount={1} label="Label" mode={mode} style={{ width: "8rem" }}>
+          {column === "with-content" && <OnyxIcon icon={mockPlaywrightIcon} />}
+        </GridElement>
+      ),
+      beforeScreenshot: async (component, page, column, row) => {
+        if (row === "hover") await component.hover();
+        if (row === "focus-visible") await page.keyboard.press("Tab");
+        if (row === "active") await page.mouse.down();
```

Comment:
We don't have a design for active state so I aligned it with the OnyxButton. There the active color is the same as the default one so the background will change on hover, an when clicking/active it will revert back to the default background

Reply:
I did not question your implementation, just that I was wondering in the screenshot why the active one did not also have the hover outlines. so I was wondering if this page.mouse.down actually does anything (or if you just press anywhere on the page, not on the comopnent)

Reply:
-> 
![image](https://github.com/user-attachments/assets/97dc5c84-92a4-405a-ad97-7ab35074a289)

Reply:
Aah now I know what you mean. Good catch! I fixed it